core/docs/ApiChangelogs/0.1.0-0.2.0.txt

```diff
@@ -0,0 +1,1471 @@
+[C] : Signature changed
+[O] : Old signature
+[N] : New signature
+[D] : Deprecated
+
+---------------------------------
+API diff
+---------------------------------
+
+[C] Streamly.Unicode.Parser
+    [A] double :: Monad m => Parser Char m Double
+[C] Streamly.FileSystem.Handle
+    [A] readWith :: MonadIO m => Int -> Handle -> Stream m Word8
+    [A] readChunksWith :: MonadIO m => Int -> Handle -> Stream m (Array Word8)
+[C] Streamly.FileSystem.Dir
+    [C] readEither
+        [O] readEither :: MonadIO m => FilePath -> Stream m (Either FilePath FilePath)
+        [N] readEither :: (MonadIO m, MonadCatch m) => FilePath -> Stream m (Either FilePath FilePath)
+    [C] read
+        [O] read :: MonadIO m => FilePath -> Stream m FilePath
+        [N] read :: (MonadIO m, MonadCatch m) => FilePath -> Stream m FilePath
+[C] Streamly.Data.Unfold
+    [A] class Enum a => Enumerable a
+    [A] second :: b -> Unfold m (a, b) c -> Unfold m a c
+    [A] first :: a -> Unfold m (a, b) c -> Unfold m b c
+    [A] enumerateFromTo :: (Enumerable a, Monad m) => Unfold m (a, a) a
+    [A] enumerateFromThenTo :: (Enumerable a, Monad m) => Unfold m (a, a, a) a
+    [A] enumerateFromThen :: (Enumerable a, Monad m) => Unfold m (a, a) a
+    [A] enumerateFrom :: (Enumerable a, Monad m) => Unfold m a a
+[C] Streamly.Data.StreamK
+    [C] parseChunks
+        [O] parseChunks :: (Monad m, Unbox a) => ParserK a m b -> StreamK m (Array a) -> m (Either ParseError b)
+        [N] parseChunks :: (Monad m, Unbox a) => ParserK (Array a) m b -> StreamK m (Array a) -> m (Either ParseError b)
+    [C] parseBreakChunks
+        [O] parseBreakChunks :: (Monad m, Unbox a) => ParserK a m b -> StreamK m (Array a) -> m (Either ParseError b, StreamK m (Array a))
+        [N] parseBreakChunks :: (Monad m, Unbox a) => ParserK (Array a) m b -> StreamK m (Array a) -> m (Either ParseError b, StreamK m (Array a))
+    [A] parseBreak :: forall m a b. Monad m => ParserK a m b -> StreamK m a -> m (Either ParseError b, StreamK m a)
+    [A] parse :: Monad m => ParserK a m b -> StreamK m a -> m (Either ParseError b)
+    [A] handle :: (MonadCatch m, Exception e) => (e -> m (StreamK m a)) -> StreamK m a -> StreamK m a
+    [A] bracketIO :: (MonadIO m, MonadCatch m) => IO b -> (b -> IO c) -> (b -> StreamK m a) -> StreamK m a
+[C] Streamly.Data.Stream
+    [A] wordsBy :: Monad m => (a -> Bool) -> Fold m a b -> Stream m a -> Stream m b
+    [A] splitOn :: Monad m => (a -> Bool) -> Fold m a b -> Stream m a -> Stream m b
+    [C] handle
+        [O] handle :: (MonadCatch m, Exception e) => (e -> Stream m a) -> Stream m a -> Stream m a
+        [N] handle :: (MonadCatch m, Exception e) => (e -> m (Stream m a)) -> Stream m a -> Stream m a
+    [A] groupsOf :: Monad m => Int -> Fold m a b -> Stream m a -> Stream m b
+[C] Streamly.Data.ParserK
+    [D] fromParser :: (MonadIO m, Unbox a) => Parser a m b -> ParserK (Array a) m b
+    [D] fromFold :: (MonadIO m, Unbox a) => Fold m a b -> ParserK (Array a) m b
```

Comment:
If we deprecate `fromFold` we should have corresponding required parser combinators. Like, `drain`, `all`, `any` etc.
Why can't folds be converted to parsers, again?
`Fold initial step extract final` = `Parser initial step final`
Does the Fold applicative make sense anymore, given scanning is what we have in mind?

Reply:
instead of K.fromFold we use D.fromFold and `K.adapt` now.